libraries/resource_systemd_automount.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'resource_systemd_unit'
+require_relative 'systemd_automount'
+
+class Chef::Resource
+  class SystemdAutomount < Chef::Resource::SystemdUnit
+    self.resource_name = :systemd_automount
+    provides :systemd_automount
+
+    def unit_type(_ = nil)
```

Comment:
> do we need this as a method override?

we seem to? i tried using a class variable and a class instance variable, but they both exploded in the unit_path method in the helper module :/. i agree that'd be better, though. feel free to take a swing at it if you know how to beat that issue!